pyrobuf/protobuf/templates/proto_pyx.tmpl

```diff
@@ -151,23 +154,32 @@ cdef class {{ message.full_name }}:
         {%- if field.deprecated|default(false) == true %}
         warnings.warn("field '{{ field.name }}' is deprecated", DeprecationWarning)
         {%- endif %}
+        {%- for oneof in message.oneofs.values() %}
+            {%- if field.name in oneof.fields %}
+                {%- for oneof_sibling in oneof.fields %}
+                    {%- if field.name != oneof_sibling %}
+        self._clearfield("{{ oneof_sibling }}")
```

Comment:
Rather than call `_clearfield` on each sibling, I think we should push some of the work of `_clearfield` to compile time and just do the correct clearing here, but we need the `ClearField` method regardless and I am mainly concerned about maintaining serialization speed, so ultimately I'm fine with this.

Reply:
The main thing I don't like with this approach is that clearfield has to do a string match. Without restructuring much of the class (one clear method per field) or template (factor out the clear logic into a sub-template) I don't see a quick alternative

Reply:
Yeah, I also don't like the string match. Think both a clear method per field (which is sort of what I was imagining) or a sub-template are good solutions, but we can punt to a future PR. This only matters if you're actually using `oneof`, which you currently can't, and anything is better than nothing.